### PR TITLE
Remove periods after URLs in log lines.

### LINKED ISSUE #760: Do not log a period after an image URL
## Description

Airflow across all browsers will render a URL at the end of a sentence as a URL with a period at the end. I encountered this while looking into #759. In the logs, the message `[2022-10-01, 00:17:59 UTC] {requester.py:69} WARNING - Unable to request URL: https://cdn.stocksnap.io/img-thumbs/960w/MF8LSXR049.jpg.  Status code: 403` will create a URL with the value `https://cdn.stocksnap.io/img-thumbs/960w/AMEYDHBGJF.jpg.`, which will fail to resolve. However, the URL https://cdn.stocksnap.io/img-thumbs/960w/AMEYDHBGJF.jpg (note the missing period at the end) _does_ resolve. 

We should remove the period after the URL in this log line:

https://github.com/WordPress/openverse-catalog/blob/0072114ea479b378a94e1144340935023acbdf3d/openverse_catalog/dags/common/requester.py#L69-L72

Ideally, we'd also try to remove any other instances of this from the repo. A cursory search I did brought up a few more cases:

```bash
$ rg -e "url.*?}\."
archive/providers/commoncrawl/Provider.py
354:                    logging.warning('Content not found for url: {}, warc file: {}, offset: {}, length: {}.'.format(data[0].strip(), data[1].strip(), data[2].strip(), data[3].strip()))

openverse_catalog/dags/common/requester.py
70:                    f"Unable to request URL: {response.url}.  "
85:            logger.error(f"Error with the request for URL: {url}.")

openverse_catalog/dags/common/urls.py
57:            f"Invalid url {url_string}, attempted upgrade: {upgraded_url}."

openverse_catalog/dags/providers/provider_api_scripts/stocksnap.py
122:        image_url = f"{IMAGE_CDN}/{img_id}.jpg"

openverse_catalog/dags/providers/provider_api_scripts/rawpixel.py
34:                f"Unable to request URL: {url}. Status code: " f"{response.status_code}"

openverse_catalog/dags/providers/provider_api_scripts/modules/etlMods.py
158:                f"Unable to request URL: {_url}. Status code:" f"{response.status_code}"

openverse_catalog/dags/common/licenses/licenses.py
225:        raise InvalidLicenseURLException(f"Failed to rewrite {derived_url}.")

openverse_catalog/dags/common/etl/scripts/ExtractCCLinks.py
59:        self.url = "https://{}.s3.amazonaws.com/crawl-data/{}/wat.paths.gz".format(

tests/factories/github/pull.json
47:      "compare_url": "https://api.github.com/repos/WordPress/openverse-frontend/compare/{base}...{head}",
192:      "compare_url": "https://api.github.com/repos/helpful-contributor/openverse-frontend/compare/{base}...{head}",
```

Notably:

https://github.com/WordPress/openverse-catalog/blob/0072114ea479b378a94e1144340935023acbdf3d/openverse_catalog/dags/common/requester.py#L85

https://github.com/WordPress/openverse-catalog/blob/0072114ea479b378a94e1144340935023acbdf3d/openverse_catalog/dags/common/urls.py#L57

https://github.com/WordPress/openverse-catalog/blob/0072114ea479b378a94e1144340935023acbdf3d/openverse_catalog/dags/providers/provider_api_scripts/rawpixel.py#L34

https://github.com/WordPress/openverse-catalog/blob/88322d2da852324f1147417ff8da332721a9002a/openverse_catalog/dags/providers/provider_api_scripts/modules/etlMods.py#L158

https://github.com/WordPress/openverse-catalog/blob/0072114ea479b378a94e1144340935023acbdf3d/openverse_catalog/dags/common/licenses/licenses.py#L225

## Resolution

- [ ] 🙋 I would be interested in resolving this bug.

### PR DESCRIPTION
**Pull request by kamiwis**
## Fixes
Fixes #760 by @AetherUnbound

## Description
As per the Issue description, there are instances where URLs will not render properly when they occur at the end of a sentence when a period follows the URL. Periods after URLS have been removed at the following locations:

[openverse-catalog/openverse_catalog/dags/common/requester.py](https://github.com/WordPress/openverse-catalog/blob/0072114ea479b378a94e1144340935023acbdf3d/openverse_catalog/dags/common/requester.py#L85)
Line 85
 `logger.error(f"Error with the request for URL: {url}.")`

[openverse-catalog/openverse_catalog/dags/common/requester.py](https://github.com/WordPress/openverse-catalog/blob/0072114ea479b378a94e1144340935023acbdf3d/openverse_catalog/dags/common/requester.py#L69-L72)
Lines 69 to 72
 `logger.warning( 
     f"Unable to request URL: {response.url}.  " 
     f"Status code: {response.status_code}" 
 )`

[openverse-catalog/openverse_catalog/dags/common/urls.py](https://github.com/WordPress/openverse-catalog/blob/0072114ea479b378a94e1144340935023acbdf3d/openverse_catalog/dags/common/urls.py#L57)
Line 57
`f"Invalid url {url_string}, attempted upgrade: {upgraded_url}."`

[openverse-catalog/openverse_catalog/dags/providers/provider_api_scripts/rawpixel.py](https://github.com/WordPress/openverse-catalog/blob/0072114ea479b378a94e1144340935023acbdf3d/openverse_catalog/dags/providers/provider_api_scripts/rawpixel.py#L34)
Line 34
`f"Unable to request URL: {url}. Status code: " f"{response.status_code}"`

[openverse-catalog/openverse_catalog/dags/providers/provider_api_scripts/modules/etlMods.py](https://github.com/WordPress/openverse-catalog/blob/88322d2da852324f1147417ff8da332721a9002a/openverse_catalog/dags/providers/provider_api_scripts/modules/etlMods.py#L158)
Line 158
 `f"Unable to request URL: {_url}. Status code:" f"{response.status_code}"`

[openverse-catalog/openverse_catalog/dags/common/licenses/licenses.py](https://github.com/WordPress/openverse-catalog/blob/0072114ea479b378a94e1144340935023acbdf3d/openverse_catalog/dags/common/licenses/licenses.py#L225)
Line 225
 `raise InvalidLicenseURLException(f"Failed to rewrite {derived_url}.")`

## Checklist

- [x] My pull request has a descriptive title (not a vague title like `Update index.md`).
- [x] My pull request targets the *default* branch of the repository (`main`) or a parent feature branch.
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [ ] I added or updated tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no visible errors.

[best_practices]:https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

File: openverse_catalog/dags/common/licenses/licenses.py
Comment (dhruvkb) [inline openverse_catalog/dags/common/licenses/licenses.py:225]: Just to maintain parity with the other similar log lines.
```suggestion
        raise InvalidLicenseURLException(f"Failed to rewrite URL: {derived_url}")
```

File: openverse_catalog/dags/providers/provider_api_scripts/rawpixel.py
Comment (dhruvkb) [inline openverse_catalog/dags/providers/provider_api_scripts/rawpixel.py:34]:
```suggestion
                f"Unable to request URL: {url} Status code: {response.status_code}"
```

**Review by dhruvkb — APPROVED** [review]
LGTM, except with some non-blocking suggestions!

**Comment by dhruvkb** [comment]
Thanks @kamiwis, this is a great fix. As we await a second review, feel free to pick any other issue in the meantime!

File: openverse_catalog/dags/providers/provider_api_scripts/rawpixel.py
Comment (kamiwis) [reply openverse_catalog/dags/providers/provider_api_scripts/rawpixel.py:34]: These are great suggestions, thank you!

**Review by AetherUnbound — APPROVED** [review]
This is great, thanks for the PR @kamiwis and the additional improvements @dhruvkb! 😄 Congrats on your first contribution to Openverse! 🎉

**Pull request merged**
